src/components/Hub/FilterButton.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react"
+import styled from "@emotion/styled"
+import { Button } from "@material-ui/core"
+import { Filter } from "../../redux/hub/hub.types"
+
+type CheckboxLabelProps = {
+  selected: boolean
+  highlightColor?: string
+}
+
+const ToggleButton = styled(Button)`
+  background-color: ${(props: CheckboxLabelProps) =>
+    props.selected ? '#21A6A6' : '#E6F5F5'};
```

Comment:
Should we encode these colour values in some central location?

Reply:
Maybe call this colour 'jina-yellow' or something to make changing this colour specifically easy in the future